### PR TITLE
Cloud 638 dashboard configmap

### PR DESCRIPTION
**Pull request by lee-baines**
Removed the import-dashboard k8s job.
Removed the import-dashboard docker folder.
Added configmap to forgerock-metrics helm chart which contains FR Grafana dashboards which are automatically picked up by the Grafana watcher.
Added format-grafana-dashboard.sh script to bin folder to prepare dashboards for import.
Updated README.

### CODE REVIEW

File: bin/format-grafana-dashboards.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# This script formats exported Grafana dashboard json files so they are compatible for import into
```
Comment (dgoldssfo) [inline bin/format-grafana-dashboards.sh]:
```suggestion
# This script formats exported Grafana dashboard JSON files so they are compatible for import into 
```

File: bin/format-grafana-dashboards.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# This script formats exported Grafana dashboard json files so they are compatible for import into 
+# Grafana as part of the Helm release.
+#
+# PLEASE NOTE!! this script is specific for the currently released ForgeRock dashboards.  
+# New dashboards may required some modifications to this script. The Prometheus datasource is often configured as a
```
Comment (dgoldssfo) [inline bin/format-grafana-dashboards.sh]:
```suggestion
# New dashboards may required some modifications to this script. The Prometheus data source is often configured as a 
```

File: bin/format-grafana-dashboards.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# This script formats exported Grafana dashboard json files so they are compatible for import into 
+# Grafana as part of the Helm release.
+#
+# PLEASE NOTE!! this script is specific for the currently released ForgeRock dashboards.  
+# New dashboards may required some modifications to this script. The Prometheus datasource is often configured as a 
+# variable.  Look for this variable in any new dashboards and add an equivalent sed command for to replace it with Prometheus.
```
Comment (dgoldssfo) [inline bin/format-grafana-dashboards.sh]:
```suggestion
# variable.  Look for this variable in any new dashboards and add an equivalent sed command for to replace it with Prometheus.
```

Pls fix "sed command for to replace it with..."

File: bin/format-grafana-dashboards.sh [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+# A folder containing unprepared exported dashboards from Grafana in JSON format.
+# Must be different from PROCESSED_DIR
+DASH_DIR='/path/to/dashboards'   
+
+# Folder for processed dashboards.
+PROCESSED_DIR='../helm/forgerock-metrics/dashboards' 
+
+# Overwrites dashboards of the same name in Grafana.
```
Comment (dgoldssfo) [inline bin/format-grafana-dashboards.sh]:
```suggestion
# Overwrites dashboards that have the same name in Grafana.
```

File: bin/format-grafana-dashboards.sh [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+OVERWRITE=true 
+
+# Loop through dashboards in dashboards/.  The following alterations are needed to allow the dashboards to be accepted by Grafana:
```
Comment (dgoldssfo) [inline bin/format-grafana-dashboards.sh]:
```suggestion
# Loop through dashboards in the dashboards directory.  The following changes are needed to allow the dashboards to be accepted by Grafana:
```

File: bin/format-grafana-dashboards.sh [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+# 1 - Wrap json content in a dashboard: object
```
Comment (dgoldssfo) [inline bin/format-grafana-dashboards.sh]:
```suggestion
# 1 - Wrap JSON content in a dashboard: object
```

File: bin/format-grafana-dashboards.sh [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+# 2 - Add overwrite option after dashboard content to allow to overwrite current dashboards. Set to true
+# 3 - API import doesn't recognise the datasource variables. The values are replaced with Prometheus.
```
Comment (dgoldssfo) [inline bin/format-grafana-dashboards.sh]:
```suggestion
# 3 - API import doesn't recognize the datasource variables. The values are replaced with `prometheus`.
```

File: bin/format-grafana-dashboards.sh [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+# 4 - Passed into jq command to remove __input section and output in json format.
```
Comment (dgoldssfo) [inline bin/format-grafana-dashboards.sh]:
```suggestion
# 4 - Passed into jq command to remove the __input section and output in JSON format.
```

File: helm/forgerock-metrics/README.md [hunk 9]
```diff
@@ -45,22 +45,22 @@ No restarting of Prometheus is required.
 The Grafana Helm chart is deployed as part of the kube-prometheus chart.  Grafana automatically connects to Prometheus and syncs all  
 the metrics which are visible through Graphs.  
 
-Dashboards for ForgeRock products are imported into Grafana post-deployment. A Kubernetes job mounts the dashboard files from the helm/forgerock-metrics/dashboards  
-folder, formats them to bypass a couple of limitations in the Grafana import API, and then uses the API to import them.
+Dashboards for ForgeRock products are added to the helm/forgerock-metrics/dashboards folder.  Any new dashboards must be formatted
```
Comment (dgoldssfo) [inline helm/forgerock-metrics/README.md]:
```suggestion
Dashboards for ForgeRock products are added to the helm/forgerock-metrics/dashboards folder.  Any new dashboards must be formatted.
```

File: bin/format-grafana-dashboards.sh [hunk 10]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
```
Comment (wstrange) [inline bin/format-grafana-dashboards.sh]: So this is a one time, or infrequent activity to run this ?

Perhaps add a comment on when someone needs to run it

**Pull request merged**